core/src/test/scala/unit/kafka/cluster/LogTest.scala

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
I have renamed this file. Those tests have nothing to do with the Replica. I have to check if they could be relocated to another place, perhaps in UnifiedLogTest. Do you mind if I do this as a follow up? I need this PR for the next one.

Reply:
Yeah, you're right. I just did it and it was simpler than what I though.